### PR TITLE
Update tests for simba JDBC driver

### PR DESCRIPTION
**Pull request by kokosing**

### CODE REVIEW

File: presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/PreparedStatements.java [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+import static java.sql.JDBCType.VARBINARY;
+import static java.sql.JDBCType.VARCHAR;
+
+public class PreparedStatements
+        extends ProductTest
+{
+    private static final Logger LOGGER = Logger.get(PreparedStatements.class);
+    private static final String TABLE_NAME = "textfile_all_types";
+    private static final String TABLE_NAME_MUTABLE = "all_types_table_name";
+    private static final String insertSql = "INSERT INTO %s VALUES(?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)";
```
Comment (sopel39) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/PreparedStatements.java]: upper letters: `INSERT_SQL`.

File: presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/PreparedStatements.java [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+    private static final String selectStarSql = "SELECT * FROM %s";
```
Comment (sopel39) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/PreparedStatements.java]: ditto

File: presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/JdbcTests.java [hunk 3]
```diff
@@ -245,13 +250,13 @@ public void testSqlEscapeFunctions()
     public void testSessionProperties()
             throws SQLException
     {
-        final String distributedJoin = "distributed_join";
```
Comment (sopel39) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/JdbcTests.java]: why this change? Can't we keep assertions for both?

File: presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/JdbcTests.java [hunk 4]
```diff
@@ -72,20 +71,12 @@ public Requirement getRequirements(Configuration configuration)
...
-        connection = defaultQueryExecutor().getConnection();
-    }
-
     @Test(groups = JDBC)
     @Requires(ImmutableNationTable.class)
     public void shouldExecuteQuery()
             throws SQLException
     {
-        try (Statement statement = connection.createStatement()) {
+        try (Statement statement = connection().createStatement()) {
```
Comment (sopel39) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/JdbcTests.java]: why this is better?

**Comment by sopel39** [comment]
some small comments

File: presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/JdbcTests.java (hunk 3 above)
Comment (kokosing) [reply presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/JdbcTests.java]: These changes look unrelated. I will remove them

File: presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/JdbcTests.java (hunk 4 above)
Comment (kokosing) [reply presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/JdbcTests.java]: I am bit a reluctant of having fields in product test classes. These variables may leak and causes product tests to be not eligible to be run in parallel.
